workers/default/tests/index.test.ts

```diff
@@ -9,29 +9,29 @@ jest.mock('amqplib');
  */
 const testEventData = {
   projectId: '5d2213fa0f290c0144a69c46',
-  catcherType: 'errors/nodejs',
+  catcherType: 'errors/default',
```

Comment:
so php, python and other catchers will send `errors/default`? How we can determine a Cather type then?

Reply:
they still will send event to `errors/***` according to its type